chips/nrf51/src/uart.rs

```diff
@@ -183,6 +183,24 @@ impl UART {
         }
     }
 
+    pub fn send_byte(&self, byte: u8) {
```

Comment:
make this unsafe to make sure capsules don't use it (it means you'll have to wrap the call to it in an `unsafe` block in the `io` module, but that's fine.